tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -184,20 +184,255 @@ public interface IFoo
                 .x(() => exception.Should().BeAnExceptionOfType<InvalidOperationException>());
         }
 
+        [Scenario]
+        public static void CallToObjectOnNonFake(
+            BaseClass notAFake,
+            Exception exception)
+        {
+            "Given an object that is not a fake"
+                .x(() => notAFake = new BaseClass());
+
+            "When I start to configure the object"
+                .x(() => exception = Record.Exception(() => A.CallTo(notAFake)));
+
+            "Then it throws an argument exception"
+               .x(() => exception.Should().BeAnExceptionOfType<ArgumentException>()
+                   .And.Message.Should().Contain("The specified object is not recognized as a fake object."));

```

Comment:
I'm fine with them being in resources for the production code, but they should be spelled out in the tests (or centralised in a separate resource for the tests) otherwise you are just testing whatever the production code is written to do.